nf/rust-policer/Makefile

```diff
@@ -1,6 +1,9 @@
 NO_DEFAULT_TARGETS := true
 
+$(HOME)/.cargo/bin/xargo:
+	cargo install xargo
+
 # TODO strip symbols
-compile:
-	@xargo build --release
+compile: $(HOME)/.cargo/bin/xargo
```

Comment:
note that I used xargo to get some features that weren't available at the time for a completely stripped-down binary, idk if it's actually necessary now

Reply:
now using cargo directly